entries/g/getresponse.com.json

```diff
@@ -0,0 +1,18 @@
+{
+  "GetResponse": {
+    "domain": "getresponse.com",
+    "tfa": [
+      "sms",
+      "email",
+      "totp",
+      "custom-software"
+    ],
+    "custom-software": [
+      "GetResponse App"
+    ],
+    "documentation": "https://help.elevenlabs.io/hc/articles/21636341747985",
```

Comment:
```suggestion
    "documentation": "https://www.getresponse.com/help/how-to-use-two-factor-authentication-in-getresponse.html",
```

Reply:
Sorry, too many submissions at once so I got confused here. 😢 @hkamran80

Reply:
No worries! Thank you for all your contributions!